elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/validator/DynamicConfigValidator.java

```diff
@@ -53,11 +53,28 @@
     private static final String SQL_SPLIT_REGEX = "\\s+";
     private static final String SEMI_COLON = ";";
     private static final Pattern HANDLEBAR_REGEX = Pattern.compile("<%(.*?)%>");
+    private static final String RESOURCES = "resources";
+    private static final int RESOURCE_LENGTH = 10; //"resources/".length()
+    private static final String CLASSPATH_PATTERN = "classpath*:";
+    private static final String FILEPATH_PATTERN = "file://";
+    private static final String HJSON_EXTN = "*.hjson";
 
     private ElideTableConfig elideTableConfig;
     private ElideSecurityConfig elideSecurityConfig;
     private Map<String, Object> variables;
     private String configDir;
+    private boolean configInClassPath;
+
+    public DynamicConfigValidator(String configDir) {
+        File config = new File(configDir);
+
+        if (config.exists()) {
```

Comment:
I feel like this is the best place to construct the spring resource URL with either file:// or resource://.

this.configDir value could actually be a URL containing file:// or resource:// and then used for loading security, variables, and tables (all using spring resource loader).